src/traces/scattermapbox/attributes.js

```diff
@@ -18,6 +19,48 @@ module.exports = overrideAll({
     lon: scatterGeoAttrs.lon,
     lat: scatterGeoAttrs.lat,
 
+    cluster: {
+        enabled: {
+            valType: 'boolean',
+            dflt: false,
+            description: 'Determines whether clustering is enabled or disabled.'
+        },
+        maxzoom: extendFlat({}, mapboxLayoutAtributes.layers.maxzoom, {
+            description: [
+                'Sets the maximum zoom level.',
+                'At zoom levels equal to or greater than the maxzoom, the layer will be hidden.'
+            ].join(' ')
+        }),
+        step: {
+            valType: 'number',
+            arrayOk: true,
+            dflt: -1,
+            min: -1,
+            description: [
+                'Sets the steps for each cluster.'
```

Comment:
What does this mean?

Reply:
Oh is this like if `step=5` then clusters of 5, 10, 15, 20... have different size and color (defined as arrays)?

Reply:
Yes. That's `arrayOk`.

Reply:
OK, so perhaps the description could read (assuming this is accurate):
> Sets how many points it takes to create a cluster or advance to the next cluster step. Use this in conjunction with arrays for `size` and / or `color`. If an integer, steps start at multiples of this number. If an array, each step extends from the given value until one less than the next value.